docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1884,16 +1909,14 @@ for who files taxes:
     * The reference person will submit the form, the spouse will be listed as the joint filer. 
     * There does not need to be persistence in who files jointly, it can be re-drawn each year. 
 - All other non-married simulants in a household with a W2 or 1099 will file separately, based on the income rules above (e.g., a low-income earner in a house with other earners will be randomly assigned to file or not file, independent of others in the household). Please note that simulants can BOTH be claimed as a dependent AND file their own taxes. 
-- All simulants eligible to be dependents will be assigned to a relative within the household 
-    * If there someone listed as the dependent's parents and they are filing taxes, they will be assigned to their parent 
-    * If there is not a parent, they will randomly be assigned to a tax-filing relative (not housemate or other non-relative) in the household 
-- A simulant eligble to be a dependent must: 
-    * Not be a "housemate/roommate" or "other nonrelative" to whoever is claiming them 
-    * The dependent's income must be below $4300 
-- For simulants outside of the household, they will be claimed as a dependent by their parent until age 19 OR age 24 if they are in group quarters for college 
-    * For simulants born in the simulation, they should have at least one parent/guardian tracked 
-    * For simulants not born in the simulantion, information on initializing parent/guardians is being added shortly 
-    * For simulatns with more than one parent/guardian tracked, assign the dependent randomly 
+- Simulants eligible to be dependents must qualify as ONE of the following: 
+    * Be under the age of 19 (less than or equal to 18)
+    * OR be less than 24, in GQ in college, and earn less than $10,000 
+    * OR be any age, but be either a relative (NOT "housemate/roommate" or "other nonrelative") of the reference person, or be the reference person and earn less than $4300 
+- An eligible simulant will be claimed in the below order of options: 
+    * The defined "guardian" (Note: if there are two parents/guardians and they are not filing jointly, assign the simulant randomly to one) 
+    * If there is not a "guardian" they will be randomly assigned to any relative (reference person OR relative of the reference person (NOT roommate or non-relative)) or a relative of themselves (if they are the reference person) 
```

Comment:
```suggestion
    * Otherwise, if they are the reference person or a relative of the reference person (NOT roommate or non-relative) they will be randomly assigned to someone else in the household who is the reference person or a relative of the reference person (NOT roommate or non-relative)
```